src/test/java/kitchenpos/application/OrderTableServiceTest.java

```diff
@@ -0,0 +1,263 @@
+			assertThat(clearedTable.getNumberOfGuests()).isEqualTo(0);
+		}
+	}
+
+	@Nested
+	class changeNumberOfGuests {
+		@Test
+		@DisplayName("손님 수 변경 시 입력된 수가 0보다 작으면 손님 수 변경을 할 수 없다")
+		void changeNumberOfGuestsNegative() {
+			// given
+			validOrderTable.setNumberOfGuests(-1);
+
+			// then
+			assertThatExceptionOfType(IllegalArgumentException.class)
+				.isThrownBy(() -> {
+					// when
+					orderTableService.changeNumberOfGuests(validOrderTable.getId(), validOrderTable);
+				});
+		}
+
+		@Test
+		@DisplayName("손님 수 변경 시 존재하지 않는 테이블 ID로 요청하면 손님 수 변경을 할 수 없다")
+		void changeNumberOfGuestsNonExistentTable() {
+			// given
+			validOrderTable.setNumberOfGuests(5);
+
+			when(orderTableRepository.findById(validOrderTable.getId())).thenReturn(Optional.empty());
+
+			// then
+			assertThatExceptionOfType(NoSuchElementException.class)
+				.isThrownBy(() -> {
+					// when
+					orderTableService.changeNumberOfGuests(validOrderTable.getId(), validOrderTable);
+				});
+		}
+
+		@Test
+		@DisplayName("손님 수 변경 시 테이블이 사용 중이 아니면 손님 수 변경을 할 수 없다")
+		void changeNumberOfGuestsTableNotOccupied() {
+			// given
+			validOrderTable.setOccupied(false);
+			validOrderTable.setNumberOfGuests(3);
+
+			OrderTable requestOrderTable = new OrderTable();
+			requestOrderTable.setId(validOrderTable.getId());
+			requestOrderTable.setNumberOfGuests(5);
+
+			when(orderTableRepository.findById(requestOrderTable.getId())).thenReturn(Optional.of(validOrderTable));
+
+			// then
+			assertThatExceptionOfType(IllegalStateException.class)
+				.isThrownBy(() -> {
+					// when
+					orderTableService.changeNumberOfGuests(requestOrderTable.getId(), requestOrderTable);
+				});
+		}
+
+		@Test
+		@DisplayName("손님 수 변경 시 주문 테이블이 사용 중이고 요청된 손님 수가 유효하면 손님 수 변경을 할 수 있다")
+		void changeNumberOfGuestsSuccessfully() {
+			// given
+			validOrderTable.setOccupied(true);
+			validOrderTable.setNumberOfGuests(3);
+
+			OrderTable requestOrderTable = new OrderTable();
+			requestOrderTable.setId(validOrderTable.getId());
+			requestOrderTable.setNumberOfGuests(5);
+
+			when(orderTableRepository.findById(requestOrderTable.getId())).thenReturn(Optional.of(validOrderTable));
+
+			// when
+			OrderTable updatedOrderTable = orderTableService.changeNumberOfGuests(requestOrderTable.getId(),
+				requestOrderTable);
+
+			// then
+			assertThat(updatedOrderTable.getNumberOfGuests()).isEqualTo(5);
+		}
+	}
+
+	@Nested
+	class findAll {
+		@Test
+		@DisplayName("주문 테이블 데이터가 비어 있을 때 모든 주문 테이블을 조회하면 주문 테이블을 조회할 수 없다")
+		void findAllWhenEmpty() {
+			// given
+			when(orderTableRepository.findAll()).thenReturn(Collections.emptyList());
+
+			// when
+			List<OrderTable> foundTables = orderTableService.findAll();
+
+			// then
+			assertThat(foundTables).isEmpty();
+		}
+
+		@Test
+		@DisplayName("주문 테이블 데이터가 비어 있지 않을 때 모든 주문 테이블을 조회하면 모든 주문 테이블을 조회할 수 있다")
+		void findAllWhenNotEmpty() {
+			// given
+			List<OrderTable> expectedTables = Arrays.asList(new OrderTable(), new OrderTable());
```

Comment:
변경되지 않는 리스트라면 List의 `of` 메서드를 사용해볼 수 있을 것 같아요. 🙂
```suggestion
			List<OrderTable> expectedTables = List.of(new OrderTable(), new OrderTable());
```